tech/database/mongodb/about.md

```diff
@@ -28,7 +29,7 @@ Access the directory and create a file mongodb-org-*release_series*.repo
 
 ```
 $ cd /etc/yum.repos.d/
-$ sudo touch mongodb-org-4.0.repo  
+$ sudo touch mongodb.repo  
```

Comment:
This can all be done with one command, f.e.:

```sh
$ sudo nano /etc/yum.repos.d/mongodb.repo
```

But in the first place, I wonder, Why would you remove the version? The file name change does not affect anything. Are there newer (recommended) versions one could install from Amazon. Does the 4.0 still work / have you tried installing it?